web-marketplace/src/pages/Home/Home.stories.tsx

```diff
@@ -0,0 +1,122 @@
+import { ApolloClient, ApolloProvider, InMemoryCache } from '@apollo/client';
+import { toBase64 } from '@cosmjs/encoding';
+import { QueryProjectsResponse } from '@regen-network/api/lib/generated/regen/ecocredit/v1/query';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import Analytics from 'analytics';
+import { graphql, rest } from 'msw';
+import { AnalyticsProvider } from 'use-analytics';
+
+import { LedgerProvider } from 'ledger';
+import { apiUri } from 'lib/apiUri';
+import { AuthProvider } from 'lib/auth/auth';
+
+import Home from 'pages/Home';
+
+export default {
+  title: 'Registry/Home',
+  component: Home,
+};
+
+const mockedAnalytics = Analytics({});
+const mockedClient = new ApolloClient({
+  uri: `${apiUri}/graphql`,
+  cache: new InMemoryCache(),
+  defaultOptions: {
+    watchQuery: {
+      fetchPolicy: 'no-cache',
+      errorPolicy: 'all',
+    },
+    query: {
+      fetchPolicy: 'no-cache',
+      errorPolicy: 'all',
+    },
+  },
+});
+const mockedQueryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      retry: false,
+    },
+  },
+});
+
+const MockTemplate = () => (
+  <QueryClientProvider client={mockedQueryClient}>
+    <ApolloProvider client={mockedClient}>
+      <AnalyticsProvider instance={mockedAnalytics}>
+        <AuthProvider>
+          <LedgerProvider>
+            <Home />
+          </LedgerProvider>
+        </AuthProvider>
+      </AnalyticsProvider>
+    </ApolloProvider>
+  </QueryClientProvider>
+);
+
+const sanity = graphql.link(
+  `https://${import.meta.env.VITE_SANITY_PROJECT_ID}.api.sanity.io/v1/graphql/${
+    import.meta.env.VITE_SANITY_DATASET
+  }/${import.meta.env.VITE_SANITY_TAG}`,
+);
+
+export const MockedSuccess = {
+  render: MockTemplate,
+  parameters: {
+    msw: {
+      handlers: [
+        sanity.query('AllCreditClass', req => {
+          console.log('sanity AllCreditClass');
```

Comment:
this doesn't get intercepted for some reason, we instantiate a new apollo client for it instead of using the client from the provider (which is set to the regen-server /graphql) so maybe that's the reason